ganttproject/src/main/java/net/sourceforge/ganttproject/document/ProxyDocument.java

```diff
@@ -215,10 +217,12 @@ public void process(XmlProject xmlProject) {
 
         resourceHandler.process(xmlProject);
         taskHandler.process(xmlProject);
+
+        List optionsToSave = new ArrayList(myProject.getTaskFilterManager().getOptions());
```

Comment:
Add type arguments:

```suggestion
        var optionsToSave = new ArrayList<>(myProject.getTaskFilterManager().getOptions());
```

Reply:
Done

Reply:
You don't like `var`? :)

Reply:
I don't know :) But I think it can be problem then you change End but display name has old value